src/enrichment.cc

```diff
@@ -220,8 +220,10 @@ std::set<cyclus::BidPortfolio<cyclus::Material>::Ptr> Enrichment::GetMatlBids(
       Material::Ptr mat = req->target();
       double request_enrich = cyclus::toolkit::UraniumAssay(mat) ;
 
-      if (ValidReq(req->target()) && (request_enrich <= max_enrich)) {
-        Material::Ptr offer = Offer_(req->target());
+       if (ValidReq(req->target()) &&
+	  ((request_enrich < max_enrich) ||
+	   (cyclus::AlmostEq(request_enrich, max_enrich)))){
+         Material::Ptr offer = Offer_(req->target());
         commod_port->AddBid(req, offer, this);
       }
     }
```

Comment:
I ran the for loop through clang-format to fix style issues - was faster than nit-picking details.  The entire for loop should be:

```c++
    for (it = commod_requests.begin(); it != commod_requests.end(); ++it) {
      Request<Material>* req = *it;
      Material::Ptr mat = req->target();
      double request_enrich = cyclus::toolkit::UraniumAssay(mat) ;

      if (ValidReq(req->target()) &&
          ((request_enrich < max_enrich) ||
           (cyclus::AlmostEq(request_enrich, max_enrich)))) {
        Material::Ptr offer = Offer_(req->target());
        commod_port->AddBid(req, offer, this);
      }
    }
```